webui/react/src/pages/ModelRegistry.tsx

```diff
@@ -413,13 +430,24 @@ const ModelRegistry: React.FC<Props> = ({ workspace }: Props) => {
         { key: MenuKey.SwitchArchived, label: record.archived ? 'Unarchive' : 'Archive' },
       ];
 
+      if (canMoveModel({ destination: { id: workspace?.id ?? 1 } })) {
```

Comment:
This isn't permission to check, because it is about whether the visited workspace (or 1) can be a **destination**

It might be `canModifyModel` or `canDeleteModel({ model: record })` and then `canDeleteModel` checking `model.workspaceId` in rbac scenario... but probably another RBAC question

Reply:
Based on our chat, it should be ```if (canModifyModel({ model: record })) {``` unless we want special rules for OSS